package.json

```diff
@@ -57,6 +57,11 @@
     "lint-staged": "^8.2.0",
     "prettier": "^2.0.2"
   },
+  "prettier": {
+    "trailingComma": "all",
+    "proseWrap": "always",
+    "singleQuote": true
+  },
```

Comment:
I think some other tools, particularly Husky, now complain about the config being in `package.json` 😆 FWIW, @fomentia and I have been working to move all these out into respective dot files.

Reply:
Oh I didn't realize that was a thing. I don't see any references to that in the husky readme (I could be blind, but do show an example of putting the config in a package.json for monorepos is shown in the readme there) and I'm not seeing any warnings anywhere (could be due to the version of husky this project is on.

If there's a strong preference to go with separate config files, mind if tackle migrating all of the configs to separate files in a later pr and continue with adding to the package.json for now? (going to be adding eslint there)

Reply:
Please do carry on, and don't even worry about pulling things out. Just thought I'd mention it so that when you bump into integrations repos, you'll have some context. Thanks @charlieduong94 !